frontend_lib/src/component/CardPopup/CardPopup.jsx

```diff
@@ -2,8 +2,10 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import CloseButton from '../Button/CloseButton.jsx'
+import IconButton from '../Button/IconButton.jsx'
+import { translate } from 'react-i18next'
 
-// require('./CardPopup.styl') // see https://github.com/tracim/tracim/issues/1156
+require('./CardPopup.styl')
```

Comment:
This line should be commented, right? (see https://github.com/tracim/tracim/issues/1156#issuecomment-444020695)

Reply:
Indeed, my mistake
I'll change that with the test fix

Reply:
Since it's a hack, I prefer commented. The day that we decide to change it we can just search by issue number to find where we should update.